estate/__manifest__.py

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "real estate",
+    "description": "real estate renting management system",
+    "category": "Tutorials",
+    "version": "1.1",
+    "application": True,
+    "data": ["security/ir.model.access.csv", "views/views.xml", "views/menus.xml"],
+    "author": "OMKHA",
```

Comment:
```suggestion
    "author": "Odoo",
```
As you are developing for Odoo, the company is the author of the file.